impl/src/main/java/org/eclipse/parsson/JsonPatchBuilderImpl.java

```diff
@@ -45,24 +45,27 @@
  *
  * @since 1.1
  */
-public final class JsonPatchBuilderImpl implements JsonPatchBuilder {
+final class JsonPatchBuilderImpl implements JsonPatchBuilder {
```

Comment:
dtto - keep public for compatibility for classpath mode

Reply:
The same as above.